sdk/storage/Azure.Storage.Common/src/Constants.cs

```diff
@@ -114,8 +119,9 @@ internal static class Append
 
             internal static class Block
             {
-                public const int DefaultParallelUploadCount = 4; // TODO What should the value really be?  Can we get rid of it with a different dispatch algorithm? (probably yes)
+                public const int DefaultConcurrentTransfersCount = 4; // TODO What should the value really be?  Can we get rid of it with a different dispatch algorithm? (probably yes)
```

Comment:
@JohnRusk .. this is something you'll want to be looking at.

I chose 4 because once upon a time that was the limit for IE without hacking the registry.

Reply:
I was talking to John about this and he said that 4 is actually pretty low. Maybe we should up it to 8. This is to increase the amount of time it takes for uploading and at the same time the number is still low enough to be fair to other connections as well. It doesn't have to be in this PR, we can always make another PR to increase this number.